user-docs/latest/users-roles-permissions/configuring-administrator-roles.md

```diff
@@ -0,0 +1,160 @@
+
+:::
+
+::: tab Settings
+
+Settings permissions can be configured for all settings accessible from *General > Settings* from the main navigation of the admin panel: Media Library and Webhooks (*Global settings* category) and Users & Roles (*Administration panel* category, to configure the settings of the RBAC feature). Settings permissions also allow to configure access to the Plugins and Marketplace sections of the admin panel. Each setting has its own specific set of permissions.
+
+| Setting name            | Permissions |
+| ----------------------- | ----------- |
+| Media Library | <ul><li>General</li><ul><li>"Access the Media Library settings page" - gives access to Media Library settings</li></ul></ul> 👉 Path reminder to Media Library settings: <br> *General > Settings > Global Settings - Media Library* |
+| Plugins and Marketplace | <ul><li>Marketplace</li><ul><li>"Access the Marketplace" - gives access to the Marketplace</li></ul></ul><ul><li>Plugins</li><ul><li>"Install (only for dev env)" - allows to install new plugins when in a development environment</li><li>"Uninstall (only for dev env)" - allows to uninstall plugins when in a development environment</li></ul></ul> |    
+| Webhooks | <ul><li>General</li><ul><li>"Create" - allows to create webhooks</li><li>"Read" - allows to see created webhooks</li><li>"Update" - allows to edit webhooks</li><li>"Delete" - allows to delete webhooks</li></ul></ul> 👉 Path reminder to Webhook settings: <br> *General > Settings > Global Settings - Webhook* |
+| Users and Roles | <ul><li>Users</li><ul><li>"Create (invite)" - allows to create administrator accounts</li><li>"Read" - allows to see existing administrator accounts</li><li>"Update" - allows to edit administrator accounts</li><li>"Delete" - allows to delete administrator accounts</li></ul></ul><ul><li>Roles</li><ul><li>"Create" - allows to create administrator roles</li><li>"Read" - allows to see created administrator roles</li><li>"Update" - allows to edit administrator roles</li><li>"Delete" - allows to delete administrator roles</li></ul></ul> 👉 Path reminder to the RBAC feature: <br> *General > Settings > Administration Panel* | 
+
+:::
+
+::::
+
+4. Click on the **Save** button on the top right corner.
+
+### Setting custom conditions for permissions
+
+For each permission of each category, a **Settings** button is displayed. It allows to push the permission configuration further by defining additional conditions for the administrators to be granted the permission. There are 2 possible additional conditions:
+
+- the administrator must be the creator,
+- the administrator must have the same role as the creator.
+
```

Comment:
Should we add a message saying that if the user created custom conditions, they will be visible in the list.

The 2 that are listed are defaults ones.

Reply:
@lauriejim we don't currently have the conditionals documented in the developer-docs as they are still early prototype, so I don't think we should mention it yet as that documentation is only given to customers per-request.

Once we document those conditionals then we should yes.

Reply:
We talked about that with @pwizla and decided to add the custom conditions to the developer docs. Both RBAC docs should be released around the same time, so I'm going to add a note now to explain that there can be more custom conditions than the 2 default ones.